pkg/graph/schema.graphql

```diff
@@ -369,6 +507,7 @@ A SystemIntake instance
 """
 type SystemIntake {
   archivedAt: Time
+  businessCase: BusinessCase
```

Comment:
I didn't add a query for the business case directly yet, but you can access it on the system intake it belongs to.